Menu/Tab/index.jsx

```diff
@@ -82,6 +84,16 @@ export default React.createClass({
     }
   },
 
+  componentWillUnmount () {
+    window.removeEventListener('resize', this.resizeListener)
+  },
+
+  resizeListener () {
+    window.requestAnimationFrame
+      ? window.requestAnimationFrame(() => update(this))
```

Comment:
That is correct. You have to do something like this instead (untested, but you get the idea):

```
const whenRendering = (fn) => {
  let scheduled = false
  
  return (...args) => {
    if (!scheduled) {
      requestAnimationFrame(() => {
        fn(...args))
        scheduled = false
      }
      scheduled = true
    }
  } 
}

// in `componentWillMount`
window.addEventListener('resize', whenRendering(() => update(this)))
```